doc/sphinx/source/howto/distribute-train.rst

```diff
@@ -1,17 +1,28 @@
 .. howto/distribute-train.rst 
 
 
-Train using multiple nGraph CPU backends with data parallel 
-===========================================================
+Distribute training across multiple nGraph backends 
+===================================================
 
 In the :doc:`previous section <../howto/derive-for-training>`, we described the 
 steps needed to create a "trainable" nGraph model. Here we demonstrate how to 
 train a data parallel model by distributing the graph to more than one device.
 
-As of release version 0.12, the default build is with OpenMPI.  To use the 
-`Intel MLSL`_ library, set the following compilation flag at build time: 
+These options are currently supported for available backends; choose the best one 
+for your scenario.  
 
-``-DNGRAPH_DISTRIBUTED_ENABLE=TRUE``.
+* Use ``-DNGRAPH_DISTRIBUTED_OMPI_ENABLE=TRUE`` to enable distributed training with 
+  OpenMPI. Use of this flag requires that OpenMPI be a pre-existing library 
+  in the system. If it's not present on the system, install Open MPI version 
+  2.1.1 or greater before running the compile. 
+
+* Use ``-DNGRAPH_DISTRIBUTED_MLSL_ENABLE=TRUE`` to enable the option for 
+  :abbr:`Intel® Machine Learning Scaling Library (MLSL)` for Linux* OS:
+
+  .. important:: The Intel® MLSL option applies to Intel® Architecture CPUs 
+     (``CPU``) and ``Interpreter`` backends only. For all other backends, 
+     ``OpenMPI`` is presently the only supported option. We recommend the 
+     use of `Intel MLSL` for CPU backends.
```

Comment:
Since the MPI library is needed to be downloaded - but MLSL isn't - need to mention that here to avoid confusion.

Reply:
Done